app/util/data_preparation/bamboo_prepare_data.py

```diff
@@ -1,11 +1,11 @@
-import urllib3
+import warnings
 
 from prepare_data_common import __generate_random_string, __write_to_file
 from util.api.bamboo_clients import BambooClient
 from util.conf import BAMBOO_SETTINGS
 from util.project_paths import BAMBOO_BUILD_PLANS, BAMBOO_USERS
 
-urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+warnings.filterwarnings('ignore', message='Unverified HTTPS request')
```

Comment:
could you pls check if this warning filter could be set on `prepare_data_common` level to reduce code duplication?